### PR TITLE
Hello Microvers Project Pull Request 

### PR DESCRIPTION
**Pull request by mrkamin**
It is a simple html and css project

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: aqua;
+}
```
Comment (Bondok6) [inline style.css:3]: - [x] Great job here 👌💯 but according to the requirements, you should use a **class** to select the `<h1>` tag, So Kindly add a class name to this element `<h1 class="header">` in your HTML and use it in your CSS file 
`.header {color: aqua;}`👌


image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (Bondok6) [inline index.html:7]: - [OPTIONAL] I think it would be better to use a descriptive title 💡

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @mrkamin 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅


## Required Changes ♻️

- [x] Your `README` file should be descriptive - all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present 👌
Kindly, update it 💯

image

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Felix45) [inline style.css:3]: As mentioned by the previous reviewer please consider using a `class selector` to select the `h1` tag, as described in the project requirements shown below :point_down:  this is a requirement for this project.

Screenshot from 2022-09-22 19-55-52

You can do this as shown below
#### index.html
```HTML
<h1 class="header"> Hello Microverse </h1>
```
#### style.css
```CSS
.header { color: aqua; }
```

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @mrkamin , 

Great work on making the changes requested by a previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights :heavy_check_mark: 
- No linter errors :white_check_mark: 
- Correct Github flow :white_check_mark: 
- Your repo is professional :white_check_mark: 


 ## Required Changes ♻️
_Check the comments under the review._
- [x] As mentioned by the previous reviewer, please consider updating your readme to be more descriptive and professional, you can use this [>> readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) from Microverse. A good readme file should be professional and descriptive of the project, for people viewing your repository. Please try as much as possible to fill all sections of the readme file.

Current readme
Screenshot from 2022-09-22 19-50-32


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (roshan-bajgain) [inline style.css:3]: - [ ] As requested by the previous reviewer please use the class on the `h1` tag and give style. ie:
`<h1 class="header"> Hello Microverse </h1>`
`header { color: aqua; }`

**Review by roshan-bajgain — COMMENTED** [review]
Hi @mrkamin, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**